src/hotspot/share/runtime/globals.hpp

```diff
@@ -247,6 +247,9 @@ const int ObjectAlignmentInBytes = 8;
           "page size for the environment as the maximum)")                  \
           range(0, max_uintx)                                               \
                                                                             \
+  product(bool, EnableAllLargePageSizes, false, EXPERIMENTAL,               \
+          "Using high time resolution (for Windows 11+ and Win64 only)")    \ 
```

Comment:
is this the correct description?  How does high time resolution relate to large pages?

Reply:
updated